frontend/src/main/webapp/WEB-INF/config.sample.properties

```diff
@@ -114,6 +114,9 @@ admin.permission=administrator
 #CAS logout URL
 #cas.logout.url=https://192.168.1.8:8443/cas2/logout?service=http%3A%2F%2F192.168.7.42%3A18080%2Funifiedviews%2F
 
+#in case this property is true you can define relative address in property cas.security.check.url
+#behindProxy=false
```

Comment:
Shouldn't we call it rather cas.behindProxy? Or do you foresee any use of such property in non-CAS use case?